frontend/public/locales/en/address-validation.json

```
@@ -13,18 +13,18 @@
     "dialog": {
       "address-suggestion": {
         "address-selection-legend": "Please select an address option below:",
-        "cancel-button": "Cancel",
-        "description": "We found a suggested correction for the address you entered.",
+        "cancel-button": "Back",
+        "description": "There's a problem with the address you provided. Please select the address you want to use.If you live in an apartment or suite, make sure to include that information.",
```

Even though is for the proof of concept,

```suggestion
        "description": "There's a problem with the address you provided. Please select the address you want to use. If you live in an apartment or suite, make sure to include that information.",
```